code/src/index.css

```diff
@@ -1,13 +1,216 @@
+  height: 50px;
+}
+
+.filter-btn {
+  margin: 10px;
+  padding: 8px;
+  background-color: #fff;
+  border: none;
+  border-radius: 5px; 
+  font-weight: bold;
+  transition: 0.1s ease-in;
+}
+
+.filter-btn:hover {
+  background-color: red;
+  color: #fff;
+}
+
+.movie-container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.popular-movie {
+  position: relative;
+  color: #fff;
+}
+
+.movie-img {
+  width: 100%;
+}
+
+.movie-details {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 10px;
+  background: rgba(0,0,0,.75);
+  opacity: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.movie-details:hover {
+  opacity: 1;
+}
+
+.movie-title {
+  margin-top: auto;
+  margin-bottom: 0;
+  font-size: 35px;
+}
+
+/*** SECOND PAGE STYLING ***/
+.detail-background {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+}
+
+.movie-background-img {
+  width: 100%;
+  height: 100%;
+} 
+
+.detail-information {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-image: linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%);
+}
+
+.home-page-link {
+  color: #fff;
+  text-decoration: none;
+  font-weight: bold;
+  position: absolute;
+  display: inline-flex;
+  left: 50px;
+  top: 50px;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.other-page-link {
+  color: #fff;
+  text-decoration: none;
+  font-weight: bold;
+  position: absolute;
+  display: inline-flex;
+  left: 50px;
+  top: 100px;
 }
+
+.link-title:hover {
+  animation: move 0.5s forwards 1;
+  position: relative;
+}
+
+@keyframes move {
+  from {left: 0px;}
+  to {left: 10px;}
+} 
+
+.poster-img {
+  border: 5px solid #fff;
+}
+
+.header-rating {
+  display: inline-flex;
+  margin-bottom: -20px;
+}
+
+.rating {
+  color: red;
+  font-weight: 300;
+  margin-left: 15px;
+}
+
+.error{
+  margin-top: 100px;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  color: white;
+}
+
+.error img{
+  width: 40%;
+}
+
+.error-btn {
+  padding: 12px;
+  background-color: #fff;
+  border: none;
+  border-radius: 5px;
+  font-size: 14px;
+}
+
+.loader {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 300px;
+  width: 100%;
+}
+
+/*** Other movies ***/
+.other-movies-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 200px;
+}
+
+.other-movies-title {
+  border-bottom: 2px solid red;
+}
+
+/*** RESPONSIVE ***/
+ @media (min-width: 768px) {
+  .movie-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+  
+  .detail-container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 50px;
+  }
+
+  .detail-content {
```

Comment:
Maybe you could add a detail semi-transparent background to separate text from the backdrop, which will increase readability.